### PR TITLE
Add upgradability tests for event bus

### PR DESCRIPTION
**Pull request by suleymanakbas91**
**Description**

Changes proposed in this pull request:

- Add upgradability tests for event bus

**Related issue(s)**

See #2600

### CODE REVIEW

File: tests/end-to-end/upgrade/pkg/tests/event-bus/event_bus.go [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+)
+
+const (
+	eventType           = "test-e2e"
+	subscriptionName    = "test-sub"
+	eventActivationName = "test-ea"
+	srcID               = "test.local"
+
+	success     = 0
+	fail        = 1
```
Comment (jasiu001) [inline tests/end-to-end/upgrade/pkg/tests/event-bus/event_bus.go]: success/fail constants are not used? Should them be here?

Comment (suleymanakbas91) [reply tests/end-to-end/upgrade/pkg/tests/event-bus/event_bus.go]: I removed them.

File: tests/end-to-end/upgrade/pkg/tests/event-bus/event_bus.go [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+		f.log.Infof("Create Subscriber deployment")
+		if _, err := f.k8sInterface.AppsV1().Deployments(f.namespace).Create(util.NewSubscriberDeployment(subscriberImage)); err != nil {
+			f.log.Errorf("Create Subscriber deployment: %v\n", err)
+			return err
+		}
+		f.log.Infof("Create Subscriber service")
+		if _, err := f.k8sInterface.CoreV1().Services(f.namespace).Create(util.NewSubscriberService()); err != nil {
+			f.log.Errorf("Create Subscriber service failed: %v\n", err)
+		}
+		time.Sleep(30 * time.Second)
```
Comment (jasiu001) [inline tests/end-to-end/upgrade/pkg/tests/event-bus/event_bus.go]: In future you can consider using [function](https://github.com/kyma-project/kyma/blob/master/tests/end-to-end/upgrade/pkg/waiter/waiter.go) which already exist for waiting for long term actions

**Comment by jasiu001** [comment]
LGTM

**Pull request merged**